test/python/transpiler/test_optimize_1q_decomposition.py

```diff
@@ -146,36 +144,6 @@ def test_optimize_away_idenity_no_target(self):
         result = passmanager.run(circuit)
         self.assertEqual(QuantumCircuit(1), result)
 
-    def test_optimize_error_over_target_1(self):
```

Comment:
Why are these two tests removed and none are added? Is this "XZX is re-written as ZXZ" no longer done? Or you thought the test isn't useful?

Reply:
These tests were specifically about testing the return of the internal `_error` helper function that was used to evaluate the heuristic score for a particular basis's found decomposition. That function was removed in this PR as the scoring is now all done on the Rust side and mostly integrated into the pass. I could add an equivalent test to call the rust function but it didn't seem necessary since it's already covered implicitly by the other tests.

That being said I was probably just being more lazy than I needed to, and could rewrite the tests fairly easily (I just need to convert the dags used into the format the rust function expects).